articles/multifactor-authentication/api/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Associate a New Authenticator for Use with Multifactor Authentication
+description: How to associate a new authenticator for use with MFA using the new MFA API endpoints
+beta: true
+---
+
+::: warning
+This article includes documentation on features that are still under development. These features are available to customers with early access.
+:::
+
+# Associate a New Authenticator for Use with Multifactor Authentication
+
+Auth0 allows you to configure your tenant so that your end users can self-associate a new authenticator for use in multifactor authentication.
+
+In this tutorial, you'll learn how to configure self-association of a new authenticator for use in multifactor authentication. Configuring Auth0 for such process requires the following steps:
+
+1. Obtaining an MFA token
+2. Requesting authenticator association
+3. Using the authenticator to confirm association
+
+## Background
+
+Let's say that you have enabled multifactor authentication for your tenant, and you are capable of supporting more than one type of authenticator. You can then configure your authorization process so that users who log in and do not have at least one active authenticator (other than a recovery code) can self-associate a new authenticator.
```

Comment:
@dafortune I'm not sure if we want to use `self-associate`. Was it an agreed upon term? Why not just `associate` considering is always the user who associates an authenticator? I think we are using it for a 'first-time association' (what we also call 'enrollment'), but maybe we don't need to be so specific and can just use 'associate' instead.